### PR TITLE
Documentation enhancements

### PR DESCRIPTION
**Pull request by bovlb**
Various small enhancements to documentation.

### CODE REVIEW

**Review by gsaluja9 — APPROVED** [review]
This looks great. Please merge this .

File: aperturedb/ParallelQuery.py [hunk 1]
```diff
@@ -9,22 +9,22 @@
 
 from aperturedb.DaskManager import DaskManager
 from aperturedb.Connector import Connector
-from aperturedb.types import Query, Blobs, CommandResponses
+from aperturedb.types import Commands, Blobs, CommandResponses
```
Comment (gsaluja9) [inline aperturedb/ParallelQuery.py:12]: Nice, how did this not manifest while running locally.

File: aperturedb/types.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 Command = Dict[str, Any]
 Blob = bytes
-Query = List[Command]
+Commands = List[Command]  # aka Query, but that's also a class name
```
Comment (gsaluja9) [inline aperturedb/types.py:7]: ❤️

File: aperturedb/ParallelQuery.py (hunk 1 above)
Comment (bovlb) [reply aperturedb/ParallelQuery.py:12]: Unclear.  Let's talk about local Python testing when we get a chance.

**Pull request merged**